Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -344,38 +345,47 @@ bool VolumeVerifier::IsDebugSigned() const
 
 bool VolumeVerifier::ShouldHaveChannelPartition() const
 {
-  const std::unordered_set<std::string> channel_discs{
-      "RFNE01", "RFNJ01", "RFNK01", "RFNP01", "RFNW01", "RFPE01", "RFPJ01", "RFPK01", "RFPP01",
-      "RFPW01", "RGWE41", "RGWJ41", "RGWP41", "RGWX41", "RMCE01", "RMCJ01", "RMCK01", "RMCP01",
+  static constexpr std::array<std::string_view, 18> channel_discs = {
+      "RFNE01", "RFNJ01", "RFNK01", "RFNP01", "RFNW01",
+      "RFPE01", "RFPJ01", "RFPK01", "RFPP01", "RFPW01",
+      "RGWE41", "RGWJ41", "RGWP41", "RGWX41",
+      "RMCE01", "RMCJ01", "RMCK01", "RMCP01",
   };
+  assert(std::is_sorted(channel_discs.cbegin(), channel_discs.cend()));
```

Comment:
Could this be a `static_assert` instead? According to the documentation, `std::is_sorted` is `constexpr` so it should work at compile time for that.

Reply:
It's constexpr only since C++20, do you think all supported compilers support this already?

Reply:
If it's C++20, we shouldn't rely on it.

Reply:
:(

Missed that, my bad.